src/js/video.js

```diff
@@ -810,5 +810,10 @@ videojs.dom = Dom;
  */
 videojs.url = Url;
 
+/**
+ * Replaces format-time with a custom implementation
```

Comment:
can this be moved to be below where we set `formatTime` so the two things sit together?